### PR TITLE
.nsupdate command

### PR DESCRIPTION
**Pull request by LinkSoraZelda**
The command provides instructions on how to delete pending updates, how to block future updates, and fixed a grammatical error

### CODE REVIEW

File: addons/assistance.py [hunk 1]
```diff
@@ -414,6 +414,12 @@ def check_console(self, message, channel, consoles):
     async def flashdrives(self):
         """Message on flash drives on the Wii U"""
         await self.simple_embed("Some flash drives work with the Wii U, some don't. If you have read or write errors, or games crash often, you might want to try a different flash drive or hard drive")
+        
+     @commands.command()(aliases=["nxupdate"])
```
Comment (ihaveamac) [inline addons/assistance.py]: misplaced aliases= keyword argument

**Comment by LinkSoraZelda** [comment]
Fixed

**Comment by ihaveamac** [comment]
there was an indent error in the code (starting with line 418)

**Comment by LinkSoraZelda** [comment]
editing on the web version of github is a bit wonky, sorry

**Pull request merged**